JSON Schema updates (#8906)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/latest-metrics-spec.md

<LatestYamlSpecAvailability />

> :::note Studio IDE YAML validation

Reviewer: @matthewshaver this feels like it's too long for a note (also two notes in a row) and might be better served in a "Before you migrate" or "prerequisites" section along with "[Package compatibility](https://docs-getdbt-com-git-schema-update-dbt-labs.vercel.app/docs/build/latest-metrics-spec?version=1.12#package-compatibility)"

Author: It seems like an average length callout to me, but I will move it so we're not doing two in a row. I think it's important to bring special attention to this based on the phased rollout approach

> The <Constant name="studio_ide" /> validates dbt YAML using JSON Schema from the [dbt-jsonschema](https://github.com/dbt-labs/dbt-jsonschema) project. Those definitions are aligned with the <Constant name="fusion_engine" /> and apply across [<Constant name="dbt_platform" /> release tracks](/docs/dbt-versions/cloud-release-tracks), including when your development environment is still running <Constant name="core" />. The intent is to keep editor assistance and diagnostics aligned with the direction of dbt and to fix schema gaps as they are reported.

Reviewer: ```suggestion
The <Constant name="studio_ide" /> validates dbt YAML using JSON Schema from the [dbt-jsonschema](https://github.com/dbt-labs/dbt-jsonschema) project. These definitions are aligned with the <Constant name="fusion_engine" /> and apply across all [<Constant name="dbt_platform" /> release tracks](/docs/dbt-versions/cloud-release-tracks), including when your development environment is still running <Constant name="core" />.
```

> If the editor flags YAML as invalid but <Constant name="dbt" /> commands succeed in your environment, treat your runs, compile, and preview results as the source of truth. Share examples with [dbt Support](mailto:support@getdbt.com) or your account team so the schema can be updated.

Reviewer: You could also call this a troubleshooting note within the before you migrate section?
```suggestion
If the <Constant name="studio_ide" /> flags your YAML as invalid but <Constant name="dbt" /> commands succeed, trust your run results. Share examples with [dbt Support](mailto:support@getdbt.com) or your account team so the schema can be updated.
```
